CONTRIBUTING.md

```diff
@@ -29,35 +29,26 @@ golden tests files.
 
 ## Updating golden tests
 
-Because Flutter golden files are platform-specific, the process
-will depend on what platform you are using.
-
-If you are using Linux, you can update golden tests files locally just by running
+Because Flutter golden files are platform-specific, the golden files will vary slightly depending
+on what platform you are using. To avoid creating unnecessary changes and to have the golden tests
+consistent on the continuous integration tests, they should only be regenerated on Linux.
 
-```
+You can update golden tests files locally on Linux just by running
+```shell
 flutter test --update-goldens
 ```
 
-To update the golden tests on Windows or macOS, open the [sweyer.yml](.github/workflows/sweyer.yml)
-workflow and set `update_goldens` to `true`:
-
-```yml
-name: Sweyer
-on: [push, pull_request]
-
-jobs:
-  build:
-    uses: ./.github/workflows/flutter_package.yml
-    with:
-      flutter_channel: stable
-      flutter_version: 2.10.5
-      min_coverage: 0
-      update_goldens: true # set this from false to true
-```
+To update the golden tests from Windows or MacOS, run the
+[`Update Goldens`](https://github.com/nt4f04uNd/sweyer/actions/workflows/update_goldens.yml)
+workflow on GitHub **in your fork**. Unless you are a contributor, you can't run it on the main
```

Comment:
contributor -> collaborator

Reply:
By the way, you can use the ![suggestion button](https://user-images.githubusercontent.com/6966049/168483745-ced1daed-a892-4e1c-a01b-10a9bb9cd808.png) button to suggest these changes and we can then commit them right here from GitHub.